op-batcher/compressor/cli.go

```diff
@@ -15,30 +15,33 @@ const (
 )
 
 func CLIFlags(envPrefix string) []cli.Flag {
+	prefixEnvVars := func(name string) []string {
```

Comment:
Could this function be in `op-service`?

Reply:
there already is `opservice.PrefixEnvVar`. We can make that output a string-slice, but we'd still need to bind it to `envPrefix`, so it doesn't get any shorter, or we have to repeat `envPrefix` for every call.

Reply:
we didn't have a problem repeating `envPrefix`. IMO `opservice.PrefixEnvVar` should return a string slice now

Reply:
And rename it to?

Reply:
Also I don't like repeating envPrefix, seems more error prone :/

Reply:
updated to return in a slice